src/wcs2_0/src/main/java/org/geoserver/wcs2_0/response/WCSDimensionsSubsetHelper.java

```diff
@@ -1282,6 +1282,9 @@ private Filter filter(String startAttribute, String endAttribute, Comparable min
             throw new UnsupportedOperationException("Only structuredGridCoverage2DReaders are currently supported");
         }
         List<DimensionBean> dimensions = new ArrayList<DimensionBean>();
+        if (accessor == null) {
```

Comment:
This null guard is the fix for https://osgeo-org.atlassian.net/browse/GEOS-7571.